java/cuvs-java/src/main/java/com/nvidia/cuvs/BruteForceQuery.java

```diff
@@ -40,7 +40,7 @@ public class BruteForceQuery {
    * @param queryVectors 2D float query vector array
    * @param mapping      an instance of ID mapping
    * @param topK         the top k results to return
-   * @param prefilter    the prefilter data to use while searching the BRUTEFORCE
+   * @param prefilters    the prefilters data to use while searching the BRUTEFORCE
```

Comment:
```suggestion
   * @param prefilters   the prefilters data to use while searching the BRUTEFORCE
```

Let's keep this aligned with all the other lines above it, please.